core/server/fluxruntime.go

```diff
@@ -171,13 +182,19 @@ func (cs *coreServer) ListFluxCrds(ctx context.Context, msg *pb.ListFluxCrdsRequ
 }
 
 func filterFluxNamespace(nss []v1.Namespace) *v1.Namespace {
-	for _, ns := range nss {
+	var fluxSystem *v1.Namespace = nil
```

Comment:
```suggestion
	var fluxSystem *v1.Namespace
```
nil is the default value, anyway.